src/firefly/js/util/WebUtil.js

```diff
@@ -286,4 +287,42 @@ export function deepDiff(o1, o2, p) {
         }
         console.groupEnd();
     }
-}
\ No newline at end of file
+}
+
+
+/*----------------------------< COOKIES ----------------------------*/
+export function setCookie(name, value, options = {}) {
+    var str = `${encodeURIComponent(name)}=${encodeURIComponent(value)}`;
+
+    if (value == null) options.maxage = -1;
+
+    if (options.maxage) {
+        options.expires = new Date(+new Date() + options.maxage);
+    }
+
+    if (options.path) str += '; path=' + options.path;
+    if (options.domain) str += '; domain=' + options.domain;
+    if (options.expires) str += '; expires=' + options.expires.toUTCString();
+    if (options.secure) str += '; secure';
+
+    document.cookie = str;
+}
+
+export function getCookie(name) {
+    var cookies = parseCookies(document.cookie);
+    return !!name ? cookies : cookies[name];
```

Comment:
This is sample code I took in.  It basically convert a value to boolean.
The !! are usually used in a return statement where  a boolean is expected.  ie. 
`exists = (x) => !!this.x;`
The !!name is not wrong in this case.  @robyww, can you change it to:
`return name ? cookies[name] : cookies`;